ts/components/sections/landing/clients.tsx

```diff
@@ -66,10 +66,6 @@ const projects: ProjectLogo[] = [
         name: 'Lake Trade',
         imageUrl: 'images/clients/laketrade.svg',
     },
-    {
-        name: 'Veil',
-        imageUrl: 'images/clients/veil.svg',
-    },
```

Comment:
Might as well delete `images/clients/veil.svg` and `images/explore.veil.svg` too.